.gitignore

```diff
@@ -1,3 +1,5 @@
 log/
```

Comment:
You can also use [our bootstrap's .gitignore](https://github.com/Wolox/rails-bootstrap/blob/master/.gitignore) instead. It should include everything you need to ignore unless you're using gems we don't use often